src/System.Net.HttpListener/tests/HttpListenerPrefixCollectionTests.cs

```diff
@@ -12,6 +12,7 @@ namespace System.Net.Tests
     public class HttpListenerPrefixCollectionTests
     {
         [Fact]
+        [ActiveIssue(17462, TargetFrameworkMonikers.Uap)]
```

Comment:
It makes sense if they are not related to the same issue, if not it would be a game of turning off and on test classes and test methods to figure out which of them got fixed and which didn't. Thanks for the advice :)